pkg/security/resolvers/mount/resolver.go

```diff
@@ -33,8 +32,14 @@ const (
 	deleteDelayTime                      = 5 * time.Second
 	numAllowedMountIDsToResolvePerPeriod = 75
 	fallbackLimiterPeriod                = 5 * time.Second
+	redemptionTime                       = 5 * time.Second
 )
 
+type redemptionEntry struct {
```

Comment:
what does redemption mean in this context?

Reply:
Add a chance for a misordered event (within 5sec) to get a correct mount resolution